resources/sts/4.12/hypershift/sts_hcp_support_permission_policy.json

```diff
@@ -65,36 +57,6 @@
         "arn:aws:s3:::*image-registry*"
       ]
     },
-    {
-      "Effect": "Allow",
-      "Action": "ec2:CreateNetworkInsightsPath",
-      "Resource": "arn:aws:ec2:*:*:network-insights-path/*",
-      "Condition": {
-        "StringEquals": {
-          "aws:RequestTag/red-hat-managed": "true"
-        }
-      }
-    },
-    {
-      "Effect": "Allow",
-      "Action": "ec2:DeleteNetworkInsightsPath",
-      "Resource": "arn:aws:ec2:*:*:network-insights-path/*",
-      "Condition": {
-        "StringEquals": {
-          "aws:ResourceTag/red-hat-managed": "true"
-        }
-      }
-    },
-    {
-      "Effect": "Allow",
-      "Action": "ec2:DeleteNetworkInsightsAnalysis",
-      "Resource": "arn:aws:ec2:*:*:network-insights-analysis/*",
-      "Condition": {
-        "StringEquals": {
-          "aws:ResourceTag/red-hat-managed": "true"
-        }
-      }
-    },
     {
       "Effect": "Allow",
       "Action": "ec2:RunInstances",
```

Comment:
this is minor, but the indentation in this file looks a little off. would be nice to get them all consistent

Reply:
I have formatted it so the tab spaces matches the other files. Should we define a formatting guide for json files so we have an easy way to keep them uniform?